documentation/guidance/gsuite/README.md

```diff
@@ -1,21 +1,28 @@
 # G Suite Accounts
 
+![Diagram](/images/gsuite-account-diagram.svg)
+
 ## Types
 Accounts can be divided in three major categories: personal accounts, external accounts and shared (purpose-specific) accounts. Shared accounts will be created only in very specific cases where several individuals need access to a single account for access to G Suite managed resources such as Google Analytics and common file storage.
 
-### Personal
-These full-fledged accounts, which are tied to a natural person, can access all the G Suite products and, due to the implicit costs, should be only used when group mailing is not an option.
+### External accounts
```

Comment:
All these types would benefit from examples.

Reply:
Maybe a user-oriented flowchart?
![Account selection](https://user-images.githubusercontent.com/11387611/83819635-a92b1900-a6ca-11ea-8479-856a3165dbce.png)

We can also add some example "cases studies" based on actual accounts:

`info@helpfulengineering.org`: collaborative inbox
- Is a mail-only account (inherited from the old mail server)
- Might be shared by several users

`media@helpfulengineering.org`: purpose-specific account
- Needs access to services other than mail (Analytics and Drive)
- Might be shared by several users

`rjohns@helpfulengineering.org`: personal account
- Needs access to services other than mail (G Suite Admin)
- Tied to a natural person